src/components/Chat/ColorPicker.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react';
+import {CirclePicker} from 'react-color';
+interface ColorPickerProps {
+  color: string;
+  onUpdateColor: (color: string) => void;
+}
+const ColorPicker: React.FC<ColorPickerProps> = (props) => {
+  const [isActive, setActive] = useState(false);
+  return (
+    <>
+      <span className="dot" onClick={() => setActive(!isActive)} style={{color: props.color}}>
```

Comment:
`cursor: 'pointer'` would be good here! I'm starting to standardize component styling to mui's `makeStyles`, so feel free to use that.